db/migrations/20230328194902_add_update_trigger_to_tables.js

```diff
@@ -0,0 +1,23 @@
+const { onUpdateTrigger } = require('../knexfile');
```

Comment:
I new that would come up :) Mostly because the example I found for doing this used two. I thought about combining them, but it might be useful to be able to roll back the setting for all the tables while still leaving the function defined so it can be applied to individual tables.